### DISCUSSION TITLE
CRITICAL with Prowler in Azure

### DISCUSSION THREAD

**Discussion by Rodasa** [prowler-cloud/prowler#1608] [General]
Hi, could someone help me in the correct configuration to run prowler with Azure? I get the following error: [File: check.py:341] [Module: check] CRITICAL: Check 'storage_blob_public_access_level_is_disabled' was not found for the AZURE provider.
Regards!

**Comment by toniblyx** [comment]
Hi @Rodasa, thanks for let us know. We will give it a look over the next days. We may need more details of your configuration.

**Comment by jfagoagas** [comment]
Moved here: https://github.com/prowler-cloud/prowler/issues/1633
